toqito/channel_metrics/completely_bounded_trace_norm.py

```diff
@@ -64,7 +64,7 @@ def completely_bounded_trace_norm(phi: np.ndarray) -> float:
     a_var = cp.bmat([[y0, -phi], [-phi.conj().T, y1]])
     constraints += [a_var >> 0]
     objective = cp.Minimize(
-        cp.norm(cp.partial_trace(y0, dims=(dim, dim), axis=1)) + cp.norm(cp.partial_trace(y1, dims=(dim, dim), axis=1))
+        cp.norm(cp.partial_trace(y0, dims=(dim, dim), axis=1)) + cp.norm(cp.partial_trace(y1, dims=(dim, dim), axis=1))  # type: ignore
```

Comment:
temporarily ignoring this line due to https://github.com/vprusso/toqito/pull/623#issuecomment-2153186121